doc/user-guides/secure-protect-connect.md

```diff
@@ -0,0 +1,372 @@
+            }
+EOF
+```
+
+Lets test again. This time we expect a `403`
+
+```
+curl -k -H "'Host: api.${KUADRANT_ZONE_ROOT_DOMAIN}'" "https://$GATEWAY_URL/cars" -i
+
+```
+
+### Define DNSPolicy 
+(skip of you did not setup a DNS provider during the setup)
+
+Now we have our gateway protected and communications secured, we are ready to configure DNS so it is easy for clients to connect and access the APIs we intend to expose via this gateway. Note during the setup of this walk through, we setup a `DNS Provider` secret and a `ManagedZone`.
+
+```
+kubectl --context kind-kuadrant-local apply -f - <<EOF
+apiVersion: kuadrant.io/v1alpha1
+kind: DNSPolicy
+metadata:
+  name: simple-dnspolicy
+  namespace: kuadrant-system
+spec:
+  routingStrategy: simple
+  targetRef:
+    name: api-gateway
+    group: gateway.networking.k8s.io
+    kind: Gateway
+EOF
+
+kubectl wait dnspolicy simple-dnspolicy -n kuadrant-system --for=condition=ready
+```
+
+If you want to see the DNSRecord created by the this policy, execute 
+
+```
+kubectl get dnsrecord api-gateway-api -n kuadrant-system -o=yaml
+```
+
+So now we have a wildcard DNS record to bring traffic to our gateway. 
+
+Lets test again. This time we expect a `403` still as the DENY_ALL is still in effect. Notice we no longer need to set the Host header directly.
+
+
+```
+curl -k "https://api.$KUADRANT_ZONE_ROOT_DOMAIN/cars" -i
+
+```
+
+### Override the Gateway's Deny ALL `AuthPolicy`
+
+Next we are going to allow authenticated access to our toystore API. To do this we will define an `AuthPolicy` that targets the `HTTPRoute`. Note that any new `HTTPRoutes` will still be effected by the gateway level policy but as we want users to now access this API we need to override the gateway policy. For simplicity we will use an API Key to allow access, but many other options are available.
+
+Lets define an API Key for user `bob` and `alice`
+
+```
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: bob-key
+  labels:
+    authorino.kuadrant.io/managed-by: authorino
+    app: toystore
+  annotations:
+    secret.kuadrant.io/user-id: bob
+stringData:
+  api_key: IAMBOB
+type: Opaque
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: alice-key
+  labels:
+    authorino.kuadrant.io/managed-by: authorino
+    app: toystore
+  annotations:
+    secret.kuadrant.io/user-id: alice
+stringData:
+  api_key: IAMALICE
+type: Opaque
+EOF
+``` 
+
+Now we will override the `AuthPolicy` to use API Keys:
+
+```
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1beta2
+kind: AuthPolicy
+metadata:
+  name: toystore
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: toystore
+  rules:
+    authentication:
+      "api-key-users":
+        apiKey:
+          selector:
+            matchLabels:
+              app: toystore
+        credentials:
+          authorizationHeader:
+            prefix: APIKEY
+EOF            
```

Comment:
```suggestion
EOF
```
The extra whitespace behaving oddly for me in bash where the EOF doesn't trigger